test/contract/src/main.rs

```diff
@@ -194,7 +245,13 @@ pub fn main() -> i8 {
     test_partial_load_tx_hash();
     test_high_level_apis();
     test_query();
-    test_dynamic_loading();
+    unsafe {
+        let context = ContextType::new();
+        let old_context = ContextTypeOld::new();
```

Comment:
We better hold the ownership on the stack and only pass the reference to the test_xxx methods. Rust may move the memory.